cogs/sfu/sfu.py

```diff
@@ -10,3 +10,6 @@
 
 class SFU(SFUCourses, SFURoads):
     """SFU resources on Discord."""
+
+    def cog_unload(self):
+        super().cog_unload()
```

Comment:
Do we need this if we're not doing anything special in the child class, or are we being explicit?

Reply:
Did a quick test run and saw `SFURoads.cog_unload()` got invoked without this call in `SFU` so this can be removed.